openapi/nexus.json

```diff
@@ -6164,31 +6164,31 @@
           "bool"
         ]
       },
-      "FleetRoles": {
+      "FleetRole": {
         "type": "string",
         "enum": [
           "admin",
           "collaborator",
           "viewer"
         ]
       },
-      "FleetRolesPolicy": {
+      "FleetRolePolicy": {
         "description": "Client view of a [`Policy`], which describes how this resource may be accessed\n\nNote that the Policy only describes access granted explicitly for this resource.  The policies of parent resources can also cause a user to have access to this resource.",
         "type": "object",
         "properties": {
           "role_assignments": {
             "description": "Roles directly assigned on this resource",
             "type": "array",
             "items": {
-              "$ref": "#/components/schemas/FleetRolesRoleAssignment"
+              "$ref": "#/components/schemas/FleetRoleRoleAssignment"
```

Comment:
`RoleRole` is a bit silly, really the only downside. We could change the rename to make this `FleetRoleAssignment`. Only problem is it relies on `AllowedRoles` to have `Role` in there, which I assume is why it is the way it is instead.

https://github.com/oxidecomputer/omicron/blob/4c933a1119afb8a8ec27c0a88286c80794e1bbe9/nexus/src/external_api/shared.rs#L73